infrastructure/tf_backend/tf_backend.auto.tfvars

```diff
@@ -0,0 +1,2 @@
+project_name = "aaq"
```

Comment:
Question: Will the `tf_backend` resources be shared across different environments (demo, staging, etc.)?

Reply:
Answer from @yvonnegitau : it can be. For different deployments the `key` in `demo/backend.conf` should change.